machines/bleShare/scan/scanMachine.ts

```diff
@@ -1210,6 +1267,10 @@ export const scanMachine =
       },
 
       guards: {
+        showFaceAuthConsentScreen: context => {
+          return !context.showFaceAuthConsent;
+        },
```

Comment:
same here guard says showFaceAuthConsentScreen -> so expectation would be to return true.